Dockerfile

```diff
@@ -53,6 +53,7 @@ COPY internal/ internal/
 COPY pkg/ pkg/
 
 # Build
+RUN go mod download && go mod tidy
```

Comment:
To make the tidy command work we have to add thee following line above:

```
COPY mock-kubernetes-client/ mock-kubernetes-client/
```

Otherwise this will be missing. I think we should add that anyway. I open a PR for that., I guess we just missed it because we only build the manager and never run the tests inside the container. I understand that this is only for testing but in general we don't want to run `go mod tidy` in this step.

Reply:
Fixed in: https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1005